bodo/hiframes/pd_multi_index_ext.py

```diff
@@ -80,7 +92,10 @@ class MultiIndexModel(models.StructModel):
     def __init__(self, dmm, fe_type):
         members = [
             ("data", types.Tuple(fe_type.array_types)),
-            ("names", types.Tuple(fe_type.names_typ)),
+            (
+                "names",
+                types.Tuple(fe_type.names_typ),
+            ),  # TODO: Use FrozenList like Pandas
```

Comment:
Noticed some inconsistent behavior between Bodo and Pandas on the latest version

Reply:
What was the inconsistent behavior you saw in this instance? I'm just curious.

Reply:
The TODO. Pandas uses a FrozenList

Reply:
No nothing broke. FrozenList is very similar to Tuple and nothing asserts it type. I just noticed during manual testing